### DISCUSSION TITLE
Right justify

### DISCUSSION THREAD

**Discussion by TacTicTow** [tltneon/lgsl#195] [Ideas]
Is it possible to make the player numbers from the chart right justify? So instead of

100
50
0

It looks like this? 

100
  50
    0

When there are triple digits for the players count, it just looks odd. Not an issue, more for aesthetics. I have a link below to a sample of what I am talking about. Looks at the numbers from the charts embedded in banner.

https://www.saltybawls.com/lgsl/?ip=64.44.100.82&port=28015

**Comment by TacTicTow** [comment]
Never mind. I figured it out. Thanks.

**Reply by tltneon** [reply]
I will be appreciated if you do a PR with that :)

**Reply by TacTicTow** [reply]
Sorry, kind of a noob with Github, not really a fan.  I am not sure how to even do a pull request, BUT I will attach it here. Had to change the extension to TXT for that to work. You will see I added 3 lines of code and changed 3 lines of code. Hope that works for you.

The userbar.php I use on my server is heavily modified for custom banners, so I just added my right justify code to your code direct from here. 

This is really not that noticeable a thing until you have a server with maxplayers > 100, the triple digits looks odd. You can see in these examples, based on your LGSL, that the numbers next to the chart graph are now right justified, and just looks cleaner. 

https://www.saltybawls.com/lgsl/?ip=64.44.100.82&port=28015

https://www.saltybawls.com/lgsl/?ip=162.248.92.62&port=7779

[userbar.txt](https://github.com/tltneon/lgsl/files/14813305/userbar.txt)
